### ISSUE TITLE
Can't access Customize > Widgets through Voiceover

### ISSUE THREAD

**Issue by rickmgithub** [Automattic/wp-calypso#101951]
### Context and steps to reproduce

The Voiceover accessibility app has stopped working when trying to read widgets, and now requires user to enable Screen Recognition mode, which makes Voiceover harder to use with a screen reader.

Video of the issue: note some areas of Customize work at 32 seconds; at 47 seconds you can see the Widgets area does not. User has to enable Screen Recognition mode to get past this issue, then has trouble proceeding with intended edits.

Note this video is with the Jetpack mobile app; user has similar issues in a browser.

https://github.com/user-attachments/assets/7ec838cf-6cc0-42c7-8e41-6a75e11f8884

The problem was discussed by Jetpack Mobile but considered to be an issue with widgets as a whole. Please see 9522229-zd-a8c

May also be related to - https://github.com/Automattic/themes/issues/8674

### Site owner impact

Fewer than 20% of the total website/platform users

### Severity

Major

### If a workaround is available, please outline it here.

Enabling Screen Recognition seems to help people get past the broken menus, but it causes other troubles.

### Platform

Customizer and IOS and potentially other touch+voiceover devices

**Comment by supernovia** [comment]
📌 REPRODUCTION RESULTS
- Tried to replicate, but I'm less familiar with the Voiceover feature and don't know the steps / gestures required to reproduce what I'm seeing in the video.


📌 FINDINGS/SCREENSHOTS/VIDEO
- I did take time to read the ticket for more information, and view the video for more context. If you'll view it with sound on:
-- At 32 seconds or so, the initial Customize menu works just fine,  until we go into Widgets at 38.
-- By 47 seconds, we're in the Widgets area and menu items for the different widget areas can't be selected
-- At 57 seconds, user turns on Screen Recognition and is able to access the menu, but they prefer to not use Screen Recognition as it makes it harder to use with a Screen Reader Device


📌 ACTIONS
- Setting to In Triage
- Clarifying title and replication steps
- Requesting feedback in slack (will add a link soon)
